orcoursetrion/config.py

```diff
@@ -24,6 +24,12 @@
     # GitHub API URL
     ('ORC_GH_API_URL', 'https://github.mit.edu/api/v3/'),
 
+    # GitHub Committer Name
+    ('ORC_GH_NAME', 'Orcoursetrion ODL'),
+
+    # GitHub Committer Email
+    ('ORC_GH_EMAIL', 'orcoursetrion@example.com'),
```

Comment:
I believe because it was originally just a list, and later I added defaults and the lazy fix was to make them tuples.  Is there a specific case for making it a dictionary? Since it is all static, immutable, and non-duplicate tuples seem to fit about as well as dictionaries to me, but I'm open to changing it if there is a good reason